pkg/onchain/website/uploader.go

```diff
@@ -20,7 +20,7 @@ import (
 var content embed.FS
 
 const (
-	blockLength = 260000
+	blockLength = 78000
```

Comment:
why this value ? the new blockSize is 300kb
puting 78kb here will make website upload very painfull :/ (lot of chunks to upload)

Reply:
https://github.com/massalabs/station/issues/1263#issuecomment-1806512678

Reply:
how to chose the value then?